app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -121,6 +121,16 @@ data class AppSettings(
         defaultValue = "1",
     )
     val blurNSFW: Boolean,
+    @ColumnInfo(
+        name = "fab_action_mode",
```

Comment:
This one doesn't make too much sense to me, especially since we might add different actions to different pages. Since there won't be a "universal" FAB action on all pages, it probably doesn't make sense to have this.

The post page, like boost, could have a multi-action FAB, to account for extra actions.

Reply:
Sure, we could add a bottom sheet on long press to customize the FAB per page type instead.

I had looked into using something like https://github.com/leinardi/FloatingActionButtonSpeedDial but I was concerned it might not be 'material 3'-ey enough. There's a branch of it that is designed for material 3, but it seems like some other way of customizing the FAB, or changing the FAB behavior, or adding more actions would be more appropriate with a bottom sheet or menu.

Reply:
Ya, I'm surprised android doesn't already have something set up for multiple floating actions / expanding.